yt/go/bus/bus.go

```diff
@@ -22,9 +26,25 @@ const (
 	AttributeKeyFeatureName AttributeKey = "feature_name"
 )
 
+type EncryptionMode int
+type VerificationMode int
+
+const (
+	EncryptionModeDisabled EncryptionMode = 0
+	EncryptionModeOptional EncryptionMode = 1
+	EncryptionModeRequired EncryptionMode = 2
+
+	VerificationModeNone VerificationMode = 0
+	VerificationModeCa   VerificationMode = 1
```

Comment:
Abbreviations must be in single case. `Ca -> CA`

https://zchee.github.io/golang-wiki/CodeReviewComments/#initialisms

Reply:
Fixed